assets/js/modules/analytics/datastore/properties.test.js

```diff
@@ -255,6 +259,98 @@ describe( 'modules/analytics properties', () => {
 				expect( console ).toHaveErrored();
 			} );
 		} );
+
+		describe( 'getPropertiesIncludingGA4', () => {
+			it( 'returns a sorted list of ua and ga4 properties ', async () => {
+				const testAccountID = fixtures.profiles[ 0 ].accountId; // eslint-disable-line sitekit/acronym-case
+				const accountID = testAccountID;
+
+				registry.dispatch( STORE_NAME ).receiveGetProperties(
+					[
+						{
+							// eslint-disable-next-line sitekit/acronym-case
+							accountId: '151753095',
+							id: 'UA-151753095-1',
+							name: 'rwh',
+						},
+						{
+							// eslint-disable-next-line sitekit/acronym-case
+							accountId: '151753095',
+							id: 'UA-151753095-1',
+							name: 'troubled-tipped.example.com',
+						},
+
+					],
+					{ accountID }
+				);
+				registry.dispatch( MODULES_ANALYTICS_4 ).receiveGetProperties( [
+					{
+						_id: '151753095-3',
+						_accountID: '151753095',
+						displayName: 'www.elasticpress.io',
+					},
+					{
+						_id: '151753095-4',
+						_accountID: '151753095',
+						displayName: 'troubled-tipped.example.com',
+					},
+				], { accountID } );
+
+				const properties = registry.select( STORE_NAME ).getPropertiesIncludingGA4( testAccountID );
+
+				expect( properties ).toHaveLength( 4 );
+
+				expect( properties[ 0 ].id ).toBe( 'UA-151753095-1' );
+				expect( properties[ 1 ]._id ).toBe( '151753095-4' );
+				expect( properties[ 2 ].id ).toBe( 'UA-151753095-1' );
+				expect( properties[ 3 ]._id ).toBe( '151753095-3' );
+
+				expect( properties[ 0 ].name ).toBe( 'rwh' );
+				expect( properties[ 1 ].displayName ).toBe( 'troubled-tipped.example.com' );
+				expect( properties[ 2 ].name ).toBe( 'troubled-tipped.example.com' );
+				expect( properties[ 3 ].displayName ).toBe( 'www.elasticpress.io' );
+			} );
+
+			it( 'returns only ua properties when ga4 datastore not available ', async () => {
```

Comment:
@danielgent please take my apologies, but I think we don't need this test because it does nothing but repeats the test above. Let's remove this.

Reply:
No problem @eugene-manuilov 

It just tests that the code I've written doesn't explode if the ga4 datastore isn't available.  It's not where I'd expect it to fail though :+1: